static/js/locator-bundle.js

```diff
@@ -5,3 +5,7 @@ export { InteractiveMap }
 import { NewMap } from './interactive-map/NewMap.js'
 window.NewMap = NewMap;
 export { NewMap }
+
+import { CardListenerAssigner } from './interactive-map/CardListenerAssigner.js'
+window.CardListenerAssigner = CardListenerAssigner;
```

Comment:
do we need to put this on the window?

Reply:
Since this is the locator bundle which doesn't put an object on the window such as HitchhikerJS, I do think we need to put it on the window unless there is another way to access it